table.go

```diff
@@ -54,11 +54,22 @@ func (e ErrCreateSchemaWriter) Error() string {
 }
 
 type TableConfig struct {
-	schema *dynparquet.Schema
+	schema       *dynparquet.Schema
+	rowGroupSize int
+}
+
+type TableOption func(*TableConfig) error
+
+func WithRowGroupSize(numRows int) TableOption {
```

Comment:
Should this be specified in bytes? It's harder to reason about where exactly row groups get split, but other observations (e.g. expected size over the wire) are easier to do. Additionally, it seems like bytes would be more in line with the other knobs (active block size).

nit: I would also add a quick comment to describe what behavior is triggered when a row group reaches this size